src/core/unit-serialize.c

```diff
@@ -590,7 +590,7 @@ static void print_unit_dependency_mask(FILE *f, const char *kind, UnitDependency
                 { UNIT_DEPENDENCY_DEFAULT,            "default"            },
                 { UNIT_DEPENDENCY_UDEV,               "udev"               },
                 { UNIT_DEPENDENCY_PATH,               "path"               },
-                { UNIT_DEPENDENCY_MOUNTINFO_OR_FILE,  "mountinfo-or-file"  },
+                { UNIT_DEPENDENCY_NON_EXEC,           "non-exec"           },
```

Comment:
This new dependency type was introduced after the previous release. Hence, it is ok to rename it. Actually, I do not like the name `mountinfo-or-file`. Better name is welcome!

But, I think `non-exec` sounds slightly broad and still ambiguous for me...

Reply:
It's intentionally broad so it could be by other unit types too... I don't known I currently have no better idea.